.vscode/settings.json

```diff
@@ -7,12 +7,6 @@
     },
     "files.insertFinalNewline": true,
     "files.trimTrailingWhitespace": true,
-    "files.exclude": {
-        "target_ra/**": true,
-        "target_wasm/**": true,
-        "target/**": true,
-        "venv/**": true,
-    },
```

Comment:
Noooo, now these folders are gonna start showing up in search results again

Reply:
What do you mean? They shouldn't show up unless you have it open in your editor

Reply:
![image](https://github.com/rerun-io/rerun/assets/1665677/12ba5490-5cab-4e38-948c-68db57de4dc1)

![image](https://github.com/rerun-io/rerun/assets/1665677/59289d6e-780c-49e4-918a-37ddcff0a077)

Reply:
I'll revert this for now so I can merge this, but I don't think it should be necessary